include/eve/module/core/constant/eps.hpp

```diff
@@ -7,22 +7,37 @@
 //==================================================================================================
 #pragma once
 
-#include <eve/as.hpp>
-#include <eve/concept/value.hpp>
-#include <eve/detail/implementation.hpp>
-#include <eve/detail/meta.hpp>
-#include <eve/module/core/constant/constant.hpp>
-#include <eve/module/core/decorator/roundings.hpp>
-
-#include <type_traits>
+#include <eve/arch.hpp>
+#include <eve/traits/overload.hpp>
+#include <eve/module/core/decorator/core.hpp>
 
 namespace eve
 {
+template<typename Options>
+struct eps_t : constant_callable<eps_t, Options, downward_option, upward_option>
```

Comment:
Why are we supporting these oiptions when they don't do anything?

Reply:
This is the generalized epsilon defined as the smallest positive value so that `1 + eps != 1` and this definition works for integers

Reply:
where K is the mathematical value
We use that sometimes for instance for trigonometry
pi is a quite weird case because the nearest double and the nearest float are not on the same size.
It is also a hook for interval arithmetic

Reply:
You're right they are not needed if the three values are the same, but already it is sure that among these three values 2 at least are equal. For me it's orthogonality sugar for the all equal case.